services/cosi-driver-nutanix/0.0.4/defaults/cm.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudnative-pg-0.23.0-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |-
+    cosiController:
+      enabled: false # This should be deployed during k8s cluster creation by konvoy.
+    objectstorageProvisionerSidecar:
+      image:
+        registry: gcr.io
+        # We can keep this in sync with the sidecar that is deploy in CephCOSIDriver to avoid duplicate images in airgapped bundle.
+        repository: k8s-staging-sig-storage/objectstorage-sidecar
```

Comment:
What's the status of this repository considering the upcoming shutdown of Container Registry on March 18?

Reply:
We have the same problem in konvoy repo (where the cosi controller is deployed) and for now we just created https://jira.nutanix.com/browse/NCN-105277 to republish the image on ghcr.io/mesophere (open to ideas, if there is a better location).